### DISCUSSION TITLE
Client and Server mode installed in the same machine?

### DISCUSSION THREAD

**Discussion by CypressLo** [rustdesk/rustdesk#5393] [Q&A] [answered]
I've installed Rustdesk server on a PC in my home and my laptop and my brother's PC all work fine with Rustdesk client.
Recently, I need to do something on that Rustdesk PC.
I've tried to install Rustdesk client on that Rustdesk PC, but somehow the server shutdown. 
Not sure if server and client occupied the same port or not.
Did anyone try this kind of usage?
Any suggestions?

**Comment by xlionjuan** [comment]
Try run the server on Docker

**Comment by CypressLo** [comment]
Hi @xlionjuan ,

Thanks for your ideas. That's the last thing I will do. 
before that, I still want to fix it in a native way.

**Reply by SergeyMy** [reply, marked as answer]
turn off **Enable Direct IP Access**, or change the port to another
![изображение](https://github.com/rustdesk/rustdesk/assets/131688106/02a56e86-1e1b-40ec-b00e-2376b3775264)

**Comment by CypressLo** [comment]
Thanks to @SergeyMy 's solution, works fine for me now.
Appreciate it.
